src/account_id.rs

```diff
@@ -110,6 +110,11 @@ impl AccountId {
     pub fn validate(account_id: &str) -> Result<(), ParseAccountError> {
         crate::validation::validate(account_id)
     }
+
+    /// Returns the length of the underlying account id string.
+    pub fn len(&self) -> usize {
```

Comment:
Let's move it to AccountIdRef (you don't need to duplicate the implementation as there is Deref implementation) and make it `const`

Reply:
done